imagick/magick_wand_image.go

```diff
@@ -1550,6 +1550,25 @@ func (mw *MagickWand) GetImageTotalInkDensity() float64 {
 	return ret
 }
 
+func (mw *MagickWand) GradientImage(gradientType GradientType, spreadMethod SpreadMethod, startColor string, stopColor string) error {
+	ppStart := C.PixelPacket{}
+	ppStartColor := newPixelPacketFromCAPI(&ppStart)
+	ppStop := C.PixelPacket{}
+	ppStopColor := newPixelPacketFromCAPI(&ppStop)
```

Comment:
These calls to `newPixelPacketFromCAPI` don't seem neccessary, since all it does is wrap a `PixelPacket` struct around it. Then you only ever access the underlying `C.PixelPacket`

Reply:
Removed.